index.html

```diff
@@ -1,11 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Log in - Fabric</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/style.css">
 </head>
+
 <body>
-    
+    <main class="main">
+        <div class="container main-layout">
+            <section class="form-section">
+                <h1 class="title">Log in</h1>
+                <p class="subtitle">Welcome back! Please login to your account to continue</p>
+
+                <form class="form" id="form">
+                    <div class="input-field">
+                        <input type="email" name="email" class="input-field__input" id="email">
```

Comment:
А якщо ми покладемо інпут до лейбла, зможемо зберігти те як це виглядає?
Ми в цьому випадку зможемо не піклуватися про `id` які мають бути унікаьні в межах сторінки.
```suggestion
    <label class="input-field__label">
        <input type="email" name="email" class="input-field__input" />
        Your e-mail
    </label>
```